Modules/syslogmodule.c

```diff
@@ -158,20 +168,27 @@ syslog_openlog(PyObject * self, PyObject * args, PyObject *kwds)
 }
 
 
+
+/*[clinic input]
+syslog.syslog
+
+    [
+    priority: int(c_default="LOG_INFO") = LOG_INFO
+    ]
+
+    message as message_object: unicode
```

Comment:
We can use the `str` converter here.
```suggestion
    message: str
```

Reply:
Ditto for `syslog.openlog`.

Reply:
For `syslog.openlog` we need a Python object.

We can of course create a new `str` or `bytes` object, or allocate a raw memory, but it is simpler with the original argument.

Reply:
Yes, of course. Sorry for the noise.